variables.tf

```diff
@@ -22,6 +22,7 @@ variable "project_id" {
 variable "topic" {
   type        = string
   description = "The Pub/Sub topic name"
+  default     = "pubsub-topic"
```

Comment:
Why did you add this default? We definitely shouldn't have this.

Reply:
removed